pkg/adapter/awsdynamodbsource/events.go

```diff
@@ -1,32 +0,0 @@
-/*
-Copyright (c) 2020 TriggerMesh Inc.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-   http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package awsdynamodbsource
-
-import (
-	"github.com/aws/aws-sdk-go/service/dynamodbstreams"
-)
-
-// DynamoDBEvent represents an AWS Dynamo DB payload.
-type DynamoDBEvent struct {
```

Comment:
Nice! I was wondering why we needed that :)

Reply:
yep, I think we can do that for the rest of adapters as well